gahtc/mapbuilder/templates/start-map.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'website/base.html' %} 
+{% load staticfiles %} 
+
+{% block head_js_block %}
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyD97JbKx02l876FPIncmsSTS_53R353q3s" async defer></script>
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+ integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ=="
+ crossorigin=""/>
+<script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"
+ integrity="sha512-/Nsx9X4HebavoBvEBuyp3I7od5tA0UzAxs+j83KgC8PU0kgB4XiK4Lfe4y4cgBtaRJQEIFCW+oC506aPT2L1zw=="
+ crossorigin=""></script>
+ <script type="text/javascript" src="https://unpkg.com/leaflet-geosearch@2.6.0/dist/bundle.min.js"></script>
+ <script src='https://unpkg.com/leaflet.gridlayer.googlemutant@latest/Leaflet.GoogleMutant.js'></script>
+ <link  href="https://unpkg.com/leaflet-geosearch@latest/assets/css/leaflet.css" rel="stylesheet" />
+{% endblock %}
+
+{% block css_block %}
+<link rel="stylesheet" type="text/css" href="{% static 'css/main.css' %}" >
+{% endblock%}
+  
+{% block content%}
+  <div id="map"></div>
+  <script>
+  // initialize the map
+  var map = L.map('map').setView([51.505, -0.09], 13);
+  
+  // load a tile layer
+  var roads = L.gridLayer.googleMutant({
+    type: 'roadmap' // valid values are 'roadmap', 'satellite', 'terrain' and 'hybrid'
+  }).addTo(map);
+
+
+// Add GeorSearch 
+var z = window.GeoSearch;
+
+const provider = new z.GoogleProvider();
+console.log(provider, "kimani");
+
+const searchControl = new z.GeoSearchControl({
+provider:provider,
+style: 'button',
+key:'AIzaSyD97JbKx02l876FPIncmsSTS_53R353q3s',
```

Comment:
Is there a way we can pass the key and provider details form a config file? That way we can easily switch between providers when testing